src/app/docs/text-editor/text-editor-docs.component.html

```diff
@@ -4,6 +4,11 @@
     moduleSourceCodePath="src/app/public/modules/text-editor/"
     pageTitle="Text editor"
   >
+
+    <sky-alert alertType="warning">
+      This component is in beta. We do not plan to make any breaking changes to the component before the official release, but such changes are possible.
```

Comment:
Is "beta" and "release candidate" the same thing? Just asking because "release candidate" is what we've used in the past for data entry grid and some other component: https://github.com/blackbaud/skyux2-docs/blob/master/src/stache/includes/components/release-candidate-warning.html.

Reply:
@blackbaud-johnly - (Just because I saw this as it came in) - they are slightly different. A "beta" is still in a state where  though we don't anticipate breaking changes - they can happen. A "release candidate" is something that we are 99% sure that there will not be breaking changes in.

Reply:
Thanks! Just making sure we were drawing a real distinction here.